docker_test.go

```diff
@@ -122,3 +123,32 @@ func TestExternalVolume(t *testing.T) {
 	assert.Equal(t, ev2.Name, "")
 	assert.Equal(t, ev2.Provider, "")
 }
+
+func TestDockerPersistentVolume(t *testing.T) {
+	docker := NewDockerApplication()
+	container := docker.Container.Volume("/host", "/container", "RW")
+	require.Equal(t, 1, len(*docker.Container.Volumes))
+
+	pVol := (*container.Volumes)[0].SetPersistentVolume(256)
+	pVol.SetType(PersistentVolumeTypeMount)
+	pVol.SetMaxSize(128)
+	pVol.AddConstraint("cons1", "EQUAL", "tag1")
+	pVol.AddConstraint("cons2", "UNIQUE")
+
+	assert.Equal(t, 256, pVol.Size)
+	assert.Equal(t, PersistentVolumeTypeMount, pVol.Type)
+	assert.NotNil(t, pVol.MaxSize)
```

Comment:
We don't need to check for nil anymore since we've converted `MaxSize`into a non-pointer.